Client/game_sa/CPoolSAInterface.h

```diff
@@ -149,3 +149,16 @@ struct SPoolData
         }
     }
 };
+
+template <class T, class I>
+struct SVectorPoolData
+{
+    std::vector<SClientEntity<T>> entities;
+    unsigned long                 ulCount;
```

Comment:
Can `ulCount` be a `std::uint32_t` instead of `unsigned long`?

Reply:
size_t